lib/src/app/app.dart

```diff
@@ -109,4 +109,34 @@ class AppClient {
         .map((e) => Permission.values.firstWhere((element) => element.value == e))
         .toList();
   }
+
+  Future<ListOrganizationMembersResponse> listOrganizationMembers(Organization org) async {
+    final request = ListOrganizationMembersRequest()..organizationId = org.id;
+    final response = await _client.listOrganizationMembers(request);
+    return response;
+  }
+
+  Future<OrganizationInvite> createOrganizationInvite(Organization org, String email) async {
+    final request = CreateOrganizationInviteRequest()
+      ..organizationId = org.id
+      ..email = email;
+    final response = await _client.createOrganizationInvite(request);
+    return response.invite;
+  }
+
+  Future<DeleteOrganizationInviteResponse> deleteOrganizationInvite(Organization org, String email) async {
```

Comment:
instead of `DeleteOrganizationInviteResponse` I could return `void` - but that makes testing weird. Open to suggestions.

Same thing for `deleteOrganizationMember`'s return type below.

Reply:
The fact that it's empty makes me think we should actually make this `void`, since having a return type could be confusing

Reply:
For testing, instead of the condition being equality on the return type, can we test that the grpc function was called only once?

Reply:
yeah, thats a good idea, thank you! ✅